Description: Explicitly use "python3".
Forwarded: not-needed
Author: Peter Pentchev <openstack-dev@storpool.com>
Last-Update: 2017-08-06

--- a/sp-openstack
+++ b/sp-openstack
@@ -327,7 +327,7 @@
 {
 	return @search_paths if @search_paths;
 
-	my @lines = map { chomp; $_ } run_program 'python', '-c',
+	my @lines = map { chomp; $_ } run_program 'python3', '-c',
 	    'import sys; print str.join("\n", sys.path)';
 	die "Python did not report a module search path at all\n"
 	    unless @lines;
